sections/Track.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from "react"
+import { Col, Container, Row, Image, Card } from "react-bootstrap";
+import { Heading, Text } from "../components";
+import styled from "styled-components";
+
+type TrackProps = {
+  heading: string;
+  text: string[];
+  tracks: IconProps[];
+  maxWidth?: string;
+  dark?: boolean;
+};
+
+type IconProps = {
+  title: string;
+  icon: string;
+  info: string;
+  id: number;
+}
+
+const TrackWrapper = styled(Container)`
+  > * {
+    color: ${(props) =>
+      props.dark
+        ? props.theme.colors.white
+        : props.theme.colors.brand[800]} !important;
+  }
+`;
+
+const Track = ({
+  heading,
+  text,
+  tracks,
+  maxWidth = "100%",
+  dark = false,
+}: TrackProps) => {
+
+  const [activeTrack, setActiveTrack] = useState<number>(0);
+  const length = tracks.length;
+
+  return (
+    <>
+      <TrackWrapper dark={dark ? 1 : 0}>
+        <Row className="row justify-content-md-center mb-5" >
+          <Col xs={12} lg={6} className="text-center mb-4" style={{ maxWidth }}>
+            <Heading className="mb-4">{heading}</Heading>
+            {text && text.map(line => {
+              return (
+                <Text text={line} key={line}/>
+              )
+            })}
+          </Col>
+        </Row>
+        <Row className="row justify-content-md-center mb-5">
+          {tracks && tracks.map(track => {
+            return (
+              <Col xs={4} lg={2} style={{ maxWidth, opacity: track.id === activeTrack ? 1 : 0.3 }} key={track.id}>
```

Comment:
can we make the onclick be a part of the whole column instead of just the text? maybe we can add a small hover animation too (but this is more just a nice-to-have)